cmd/gobmp/gobmp.go

```diff
@@ -76,9 +76,21 @@ func main() {
 	var err error
 	switch strings.ToLower(dump) {
 	case "file":
-		publisher = filer.NewFiler(file)
+		publisher, err = filer.NewFiler(file)
+		if err != nil {
+			glog.Errorf("fail to initialize file publisher with error: %+v", err)
+			glog.Errorf("restarting gobmp...")
```

Comment:
Is this restarting or just killing?

Reply:
well, since in most of cases it runs as a deployment in kubernetes then when pod terminates, it gets restarted automatically.